BoostTestAdapter/Utility/QualifiedNameBuilder.cs

```diff
@@ -100,7 +100,7 @@ public string MasterTestSuite
         /// <summary>
         /// Identifies the default MasterTestSuite test suite name.
         /// </summary>
-        public static string DefaultMasterTestSuiteName { get { return "Master Test Suite"; } }
+        public static string DefaultMasterTestSuiteName { get { return Resources.MasterTestSuite; } }

```

Comment:
Where is this string shown?

Reply:
On closer inspection, I think this string is always stripped from the test case name before being displayed in the UI. I'll remove it from the resx.